buildbot.mariadb.org/scripts/docker_cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -xeuvo pipefail
+
+mariadb_version=${1}
+mariadb_version=${mariadb_version#*-}
+master_branch=${mariadb_version%\.*}
+
+commit=${2:-}
+branch=${3:-${master_branch}}
+
+if [[ $branch =~ ^preview ]]; then
+  container_tag=${branch#preview-[0-9]*\.[0-9]*-}
+else
+  container_tag=$master_branch
+fi
+# Container tags must be lower case.
+container_tag=${container_tag,,*}
+
+t=$(mktemp)
+for m in mariadb-devel-${container_tag}-$commit mariadb-debug-${container_tag}-$commit
+do
+  echo cleanup $m
+  buildah manifest inspect "$m" | jq '.manifests[]?.digest?' | tee -a "$t"
+done
+
+podman images --filter dangling=true --format '{{.ID}} {{.Digest}}' |
+while read line; do
+  id=${line% *}
+  digest=${line#* }
+  echo id=$id digest=$digest
```

Comment:
Not mandatory, but good habit IMO:
[shellcheck] SC2086: Double quote to prevent globbing and word splitting. [Info]